frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ClusterDetails/ClusterMachinePools/components/ScaleMachinePoolModal.tsx

```diff
@@ -135,9 +132,7 @@ export function ScaleMachinePoolModal(props: ScaleMachinePoolModalProps) {
                     required
                     label={t('machinePool.modal.scale.maxReplicas.label')}
                     id="scale-max"
-                    // We only need to enforce machineSetCount as minimum for maxReplicas
-                    // until https://issues.redhat.com/browse/HIVE-2415 is fixed
-                    min={Math.max(0, minReplicas, machineSetCount)}
+                    min={minReplicas}
```

Comment:
Minor thing:

```suggestion
                    min={Math.max(0, minReplicas)}
```

It's a little tricky, but you can enter a negative number for the minimum by typing in a number and then adding the negative sign in front. 

With your code, the user can then use the `-` button on the max replicas to go below zero.
![image](https://github.com/user-attachments/assets/aa78f9e2-43f7-4350-8273-930c764a4f99)

With the suggested code, the `-` button will be disabled when you get to 0.
![image](https://github.com/user-attachments/assets/fb3cd8d0-25e1-44f6-ac28-043f60a63b8c)

Reply:
thanks for catching that @KevinFCormier